### PR TITLE
Add partial support for 3D related conv/pooling/padding

### PR DESCRIPTION
**Pull request by jiayiliu**
Hi 
Nice work, and I found it quite useful.
I would like to push some changes for 3d conv/padding/pooling.
Not all functions are support but is sufficient for my purpose so far. 

Hope you can accept the request.
Cheers,

### CODE REVIEW

**Comment by nerox8664** [comment]
Hello @jiayiliu!

Thank you for your pull request.
I will review and test it before I can merge it, it may take some time.

File: onnx2keras/padding_layers.py [hunk 1]
```diff
@@ -13,36 +14,53 @@ def convert_padding(node, params, layers, node_name, keras_name):
...
+                name=keras_name
+            )
         layers[node_name] = padding_layer(input_0)
     elif params['mode'] == 'reflect':
 
-        def target_layer(x, pads=params['pads']):
+        def target_layer(x):
             import tensorflow as tf
-            layer = tf.pad(x, [[0, 0], [0, 0], [pads[2], pads[6]], [pads[3], pads[7]]], 'REFLECT')
+            if len(pads)==8:
```
Comment (nerox8664) [inline onnx2keras/padding_layers.py]: Please, use spaces between comparison operation. I'm about to fix all lint issues I have in this project

File: onnx2keras/padding_layers.py [hunk 2]
```diff
@@ -13,36 +14,53 @@ def convert_padding(node, params, layers, node_name, keras_name):
...
 
-        def target_layer(x, pads=params['pads']):
+        def target_layer(x):
             import tensorflow as tf
-            layer = tf.pad(x, [[0, 0], [0, 0], [pads[2], pads[6]], [pads[3], pads[7]]], 'SYMMETRIC')
+            if len(pads)==8: # TODO not tested yet
+                layer = tf.pad(x, [[0, 0], [0, 0], [pads[2], pads[6]], [pads[3], pads[7]]], 'SYMMETRIC')
+            else:
+                logger.warning("Caution - no test yet")
+                layer = tf.pad(x, [[0, 0], [0, 0], [pads[2], pads[7]], [pads[3], pads[8]], [pads[4], pads[9]]], 'SYMMETRIC')
```
Comment (nerox8664) [inline onnx2keras/padding_layers.py]: Maybe it's better not to use hard-coded constants for 3-d pad in Lambda?

File: onnx2keras/convolution_layers.py [hunk 3]
```diff
@@ -34,11 +34,91 @@ def convert_conv(node, params, layers, node_name, keras_name):
...
+                name=padding_name
+            )
+            layers[padding_name] = input_0 = padding_layer(input_0)
+        out_channels, channels_per_group, dimension, height, width = W.shape
+        W = W.transpose(2, 3, 4, 1, 0) 
+        in_channels = channels_per_group * n_groups
+
+        if n_groups != 1:
+            raise NotImplementedError("Not Implemented")
+            """
```
Comment (nerox8664) [inline onnx2keras/convolution_layers.py]: Please, delete commented block

**Comment by jiayiliu** [comment]
Changed accordingly.  I didn't run reformat the code on my end because my default setting replaces 'single quote' by "double quote".  I hope   
Meanwhile I hope I understood the pad constant review comment correctly.

**Comment by nerox8664** [comment]
Hey! Sorry for a big delay, I'll review & merge your changes soon. Once again, thank you for your work.

**Comment by nerox8664** [comment]
Merged, thank you for your work!

**Pull request merged**